Tests/MapboxNavigationTests/NavigationCameraTests.swift

```diff
@@ -611,18 +611,20 @@ class NavigationCameraTests: XCTestCase {
         let zoomRange = navigationViewportDataSource?.options.followingCameraOptions.zoomRange
         XCTAssertEqual(zoomRange?.lowerBound, 10.0, "Lower bounds should be equal.")
         XCTAssertEqual(zoomRange?.upperBound, 22.0, "Upper bounds should be equal.")
-        
-        var appliedChanges = false
-        expect {
-            // It should only be possible to set zoom range levels from `0.0` to `22.0`.
-            navigationViewportDataSource?.options.followingCameraOptions.zoomRange = -1.0...100.0
-            appliedChanges = true
-        }.to(throwAssertion())
-        
-        XCTAssertFalse(appliedChanges, "Zoom range changes should not be applied.")
-    }
-    
-    func testNavigationCameraOverviewCameraOptionsMaximumZoomLevel() {
+
+        // Disabled until https://github.com/mapbox/mapbox-navigation-ios/issues/3351 is resolved.
```

Comment:
Is it clear why this test is failing?

Reply:
yes, because `throwAssertion` function from `Nimble.Matchers.ThrowAssertion` doesn't work on ARM Macs. And this test tests the assertion.

Reply:
Have you tried to use `XCTAssertThrows` to rewrite this test without Nimble?

Reply:
Or if it doesn't work, should we just wrap the failing code with `#if arch(x86_64) or canImport(Darwin)`? (https://github.com/Quick/Nimble/blob/e4aa2014de13b90b06eaf183c67be3e390e927da/Sources/Nimble/Matchers/ThrowAssertion.swift#L9)

Reply:
I will check what's possible.

Reply:
- `XCTAssertThrows` isn't available in Swift.
- Calling `XCTAssertThrows` from Swift through ObjC doesn't work either.

Reply:
@bamx23 good alternative, thank you for idea

Reply:
`#if arch(x86_64)` doesn't work either. It enters this branch on M1 :(

Reply:
Are you running Xcode under rosetta?

Reply:
no

Reply:
it can be related to iOS 13 though.

Reply:
yes, this check doesn't work on iOS 13 but works on iOS 14 simulator.